api/src/main/java/org/pmiops/workbench/actionaudit/auditors/UserServiceAuditAdapterImpl.java

```diff
@@ -63,6 +63,7 @@ public void fireAdministrativeBypassTime(
       Optional<Instant> bypassTime) {
     ActionAuditEvent.Builder eventBuilder =
         ActionAuditEvent.builder()
+            .timestamp(clock.millis())
```

Comment:
Hmm. This means I don't have unit test coverage here. I'll add a test for this function. 

I'm glad my validator did its job though.